File: app/src/main/java/com/unsplash/client/data/model/PhotoModel.java
package com.unsplash.client.data.model;

import com.google.gson.annotations.SerializedName;

public class PhotoModel{

    @SerializedName("id")
    private String id;

    @SerializedName("description")
    private
    String description;

    @SerializedName("urls")
    private UrlsPhoto urlsPhoto;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public UrlsPhoto getUrlsPhoto() {
        return urlsPhoto;
    }

    public void setUrlsPhoto(UrlsPhoto urlsPhoto) {
        this.urlsPhoto = urlsPhoto;
    }
}

File: app/src/main/java/com/unsplash/client/data/model/PhotoModel.kt
package com.unsplash.client.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class PhotoModel {

    @SerializedName("id")
    @Expose
    var id: String? = null

    @SerializedName("description")
    @Expose
    var description: String? = null

    @SerializedName("urls")
    @Expose
    var urlsPhoto: UrlsPhoto? = null
}
